### PR TITLE
Update community.md

### PR DESCRIPTION
**Pull request by t288msd**
corrected yaml formatting for apex chart

### CODE REVIEW

File: _docs/community.md [hunk 1]
```diff
@@ -54,76 +54,73 @@ which will produce something like the following...
...
+    type: gradient
+    gradient:
+      shadeIntensity: 0.1
+      opacityFrom: 0.3
+      opacityTo: 1
+      inverseColors: true
 series:
-
-    entity: sensor.octopus_energy_electricity_***************************_current_rate
+  - entity: sensor.octopus_energy_electricity_xxxxxxxxxxxx_1100000946808_current_rate
```
Comment (BottlecapDave) [inline _docs/community.md:88]: Are you ok with this serial number to appear?

File: _docs/community.md [hunk 2]
```diff
@@ -54,76 +54,73 @@ which will produce something like the following...
...
+      duration: 5m
     stroke_width: 2
     extend_to: now
     show:
-    extremas: false
-    header_color_threshold: true
-    entity: sensor.givtcp_grid_power
+      extremas: false
+      header_color_threshold: true
+  - entity: sensor.givtcp_ce2146g163_grid_power
```
Comment (BottlecapDave) [inline _docs/community.md:102]: Are you ok for this identifier to appear?

File: _docs/community.md (hunk 1 above)
Comment (t288msd) [reply _docs/community.md:88]: thanks for picking that up. xxxx it out if you don't mind

File: _docs/community.md (hunk 2 above)
Comment (t288msd) [reply _docs/community.md:102]: thanks. missed that. xxxxx out the ce2146g163 if you don't mind.

**Comment by BottlecapDave** [comment]
Just realised I never thanked you for your fix to the documentation. Sorry about that, but thanks :)

**Comment by t288msd** [comment]
Hi David. It’s the least I can do to contribute to an excellent products you’ve developed. Thank YouSent from my iPhoneOn 24 Jun 2023, at 08:02, David Kendall ***@***.***> wrote:﻿
Just realised I never thanked you for your fix to the documentation. Sorry about that, but thanks :)

**Pull request merged**